content/en/containers/guide/operator-eks-addon.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Installing Agent on AWS EKS using Operator Add-on
+kind: guide
+aliases:
+ - /agent/guide/operator-eks-addon
+further_reading:
+  - link: 'agent/kubernetes/log'
+    tag: 'Documentation'
+    text: 'Datadog and Kubernetes'
+---
+
+Datadog Operator can be installed on EKS cluster as an add-on. Add-ons provide installation and management of a curated set of add-ons for Amazon EKS clusters. 
+
+## Prerequisites
+
+Installing Operator as an add-on requires following prerequisites:
+
+* Ability to subscribe to [Datadog Operator][1] product. 
+* [TODO] License manager SLR.
+* Kubectl, AWS CLI if using command line interface.
+
+## Installing Operator
+
+There are certain differences when using add-on, compared to the regular [Helm installation][4]:
+* During Operator installation images must be pulled only from EKS repository, this can't be change by user.
+* Operator Helm Chart values which can be overriden is restricted by following [schema file][3].
+
+These restriction are necessary to make Operator compliant with the add-on policies, allow EKS ensure safety of the installation, and disable features not yet supported in add-on environment.
+
+{{< tabs >}}
+{{% tab "Console" %}}
+
+* Go to the EKS cluster in the AWS console.
+* Go to add-on tab, select *Get more add-ons*.
+* Find *Datadog Operator*, select and follow the prompts to complete the installation.
+
+{{% /tab %}}
+{{% tab "CLI" %}}
+
+Run the following command to install Operator add-on:
+  ```bash
+  aws eks create-addon --addon-name datadog_operator --region <AWS_REGOIN> --cluster-name <CLUSTER_NAME> 
+  ```
+
+Add-on installation is asynchrnous, to check the status of add-on run:
+  ```bash
+  aws eks describe-addon --addon-name datadog_operator --region <AWS_REGOIN> --cluster-name <CLUSTER_NAME> 
+  ```
+
+To delete add-on run:
+  ```bash
+  aws eks delete-addon --addon-name datadog_operator --region <AWS_REGOIN> --cluster-name <CLUSTER_NAME>
+  ```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+To verify wheather installation was successful, confirm that datadog-operator pod is running either via Console, `eksctl` or AWS CLI.
+
+## Configuring Agent
+
+Operator add-on 0.1.x installs Operator only. For Agent setup one can follow steps 2-4 from the Operator [installation guide][2].
```

Comment:
steps 2-4 are relatively short. i think it's confusing for a user to jump to too many pages. we can just copy/paste those steps here.

Reply:
Updated in https://github.com/DataDog/documentation/pull/19482/commits/e9180c6ca31e23458b28c5ee9dea2c5542a07d06